src/helm/benchmark/static/schema.yaml

```diff
@@ -2999,12 +2991,12 @@ run_groups:
       task: question answering
       what: medical, open-domain, or truthful qa
       who: n/a
-      when: n/a
+      when: 2022 or before
       language: Chinese
 
   - name: cleva_summarization
     display_name: CLEVA (Chinese) summarization
-    description: "Contains one subtask so far: summarize a dialogue between a customer representative and a customer."
+    description: "Summarize a dialogue between a customer representative and a customer."
```

Comment:
Why are we not mentioning "one subtask so far"? I have the same question for other related scenarios as well. Thanks!

Reply:
I have no other issue with this pull request.

Reply:
This is suggested by YIfan.

Reply:
Yeah, I realize it now. Thanks.